osdp/stacks/osdp_prototype_stack.py

```diff
@@ -60,8 +58,15 @@ def __init__(
             auto_delete_objects=True,
         )
 
+        # Get ECR configuration from context
+        ecr_config = self.node.try_get_context("ecr")
+        if not ecr_config:
+            raise ValueError("ECR configuration is missing from context")
```

Comment:
@bmquinn - Could we at least not make this required when you are in EAD "mode"? 

My preference would actually be that it's not required at all - but could be present for the advanced user who wants to override it. (For the simplest use case of the less technical user they don't even need to know it exists and it will use our ECR repo by default.) But I'll leave that part up to you.

Reply:
You make a great point! I'll make sure you at least don't need this config in EAD mode. Thanks for the heads up.